src/ws-reconnect.ts

```diff
@@ -0,0 +1,51 @@
+import * as WebSocket from 'ws'
```

Comment:
hard to review this file without seeing the diff with https://github.com/interledgerjs/ilp-plugin-btp/pull/15/files#diff-96d6f42d58c5d16c4b0d812526b8426e